cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -122,6 +133,36 @@ public static DataprocClient fromConf(DataprocConf conf) throws IOException, Gen
     String subnet = conf.getSubnet();
     Network networkInfo = getNetworkInfo(projectId, network, compute);
 
+    // if a peered network is specified then the instance is tied to a network
+    if (!Strings.isNullOrEmpty(conf.getPeeredNetwork())) {
```

Comment:
I thought we talked about just checking if the configured network is peered with the CDAP network and using the internal IP if so. Are these two new properties (private instance and peered network)actually needed?